README.md

```diff
@@ -65,7 +65,7 @@ npm i vite-plugin-checker-vls -D
 
 ```js
 import Checker from 'vite-plugin-checker'
-import VlsChecker from 'vite-plugin-checker-vls'
+import {VlsChecker} from 'vite-plugin-checker-vls'
```

Comment:
```suggestion
import { VlsChecker } from 'vite-plugin-checker-vls'
```

Reply:
lgtm, it seems we have different lint style😂